buildconfig/CMake/WindowsSetup.cmake

```
@@ -59,10 +59,18 @@ if(CONDA_ENV)
   # switch off all optimzations
   set(_conda_debug_cfg_name DebugWithRelRuntime)
   string(TOUPPER ${_conda_debug_cfg_name} _conda_debug_cfg_name_upper)
+  if(CCACHE_FOUND)
```

As we have the `USE_CCACHE` option too I think we need to check that?

```suggestion
  if(USE_CCACHE AND CCACHE_FOUND)
```

as if cmake was run first with `USE_CCACHE=ON` but then run again with ``USE_CCACHE=OFF` ccache was still found but the developer no longer wants it to be used.